research/object_detection/dataset_tools/seq_example_util.py

```diff
@@ -221,7 +232,11 @@ def make_sequence_example(dataset_name,
   if image_format is not None:
     context_dict['image/format'] = context_bytes_feature([image_format])
   if image_source_ids is not None:
-    feature_list['image/source_id'] = sequence_bytes_feature(image_source_ids)
+    if (use_strs_for_source_id):
```

Comment:
why do we need the second option ?

Reply:
For whatever reason, the default way of creating the source id was making it a list of bytes which I didn't want and which TFOD didn't like during training.